src/components/ExchangePage/index.tsx

```diff
@@ -643,14 +640,15 @@ function ExchangePage({ sendingInput = false, history, params }: ExchangePagePro
     tokens,
     route,
     trade,
-    userBalances
+    userBalances,
+    account
   ])
 
   // warnings on slippage
   const warningLow: boolean =
     slippageFromTrade &&
-    parseFloat(slippageFromTrade.toFixed(4)) < ALLOWED_SLIPPAGE_MEDIUM / 100 &&
-    parseFloat(slippageFromTrade.toFixed(4)) > 0
+    parseFloat(slippageFromTrade.toFixed(4)) > 0 &&
+    parseFloat(slippageFromTrade.toFixed(4)) < ALLOWED_SLIPPAGE_MEDIUM / 100
```

Comment:
opportunity to use !greaterThan if you want to convert the constants into percents which should be more reliable

Reply:
yup, done